modules/gitops-configuring-respect-rbac-using-gitops.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-respect-rbac-using-gitops_{context}"]
+= Configuring respectRBAC using {gitops-title}
+
+The `respectRBAC` feature in Argo CD controls how Argo CD watches resources on a cluster. By default, Argo CD attempts to watch all Kubernetes resources (CRDs) on a cluster at the cluster scope. With the `respectRBAC` feature, you can restrict the ArgoCD controller from discovering or syncing specific resources using only controller RBAC, without manually configuring resource exclusions.
+
+To enable this feature, set the `.spec.controller.respectRBAC` key in the Argo CD resource. After you set this key, the controller automatically stops watching resources it cannot list or access. For example, if the Argo CD cluster role prevents Argo CD from watching OpenShift Routes, it returns an error during synchronization, stating it cannot watch the resource.
```

Comment:
How about this:
- After you set this key, the controller automatically stops watching resources it cannot list or access. For example, **_this prevents the case where_** if the Argo CD cluster role prevents Argo CD from watching OpenShift Routes, it returns an error during synchronization, stating it cannot watch the resource.
- (change in **_bold_**)

Adding 'this prevents the case where' makes it clear that enabling this feature should prevent the 'cannot watch the resource' error. WDYT?

Reply:
Fixed.